identifiers/country-au/README

```diff
@@ -3,8 +3,12 @@ README
 File contents in `country-au` directory:
 *   federal_electorates: contains electorates for the Australian House of Representatives
 *   states.csv: contains states and territories in Australia
+*   assemblies.csv: represents the district electorates and region electorates for state Legislative Assembly
+*   councils.csv: represents the district electorates and region electorates for state Legislative Council
 
 OCD ID Types used used:
 *   federal_electorate : represents the electorates for the Australian House of Representatives
 *   state: states in Australia
-*   territory: territories in Australia
\ No newline at end of file
+*   territory: territories in Australia
+*   district: districts in Australia
```

Comment:
Please update the README to reflect the new types

Reply:
Hi @jloutsenhizer, As suggested, we have updated README file.